btr-web/btr-common-components/cypress/e2e/pages/examples/form/email.cy.ts

```diff
@@ -2,6 +2,7 @@ describe('forms -> email -> validate that email component work inside example fo
   beforeEach(() => {
     // navigate to index page and check footer and header exist
     cy.visit('/examples/form')
+    cy.wait(2000)
```

Comment:
for some reason this is required now with the middleware getting invoked (I think its due to cypress not having a hydration hook or something like that)